store/rootmulti/store.go

```diff
@@ -26,11 +21,18 @@ import (
 	"github.com/cosmos/cosmos-sdk/store/transient"
 	"github.com/cosmos/cosmos-sdk/store/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	iavltree "github.com/cosmos/iavl"
```

Comment:
Please revert this block imports as they were idiomatic before

Reply:
There is a `"github.com/cosmos/cosmos-sdk/store/iavl"` on top. So we need a rename to avoid the clash.